keda/values.yaml

```
@@ -642,15 +652,29 @@ certificates:
   certManager:
     # -- Enables Cert-manager for certificate management
     enabled: false
+    # -- Certificate duration
+    duration: 8760h0m0s # 1 year
+    # -- Certificate renewal time before expiration
+    renewBefore: 5840h0m0s # 8 months
     # -- Generates a self-signed CA with Cert-manager.
     # If generateCA is false, the secret with the CA
     # has to be annotated with `cert-manager.io/allow-direct-injection: "true"`
     generateCA: true
+    # -- Generates an Issuer resource with Cert-manager
+    generateIssuer: true
+    # -- Reference to custom Issuer.
+    # If generateIssuer is false, the issuer reference below has to be provided
+    issuer:
+      {}
+      # name: foo-org-ca
+      # kind: ClusterIssuer
+      # group: cert-manager.io
```

As we use `helm-docs` and AFAIK, there isn't any way for templating values not present in the values, we need to add them somehow. I'm not sure the best approach, we can just add them (because `generateIssuer` has precedence) or we could just add it as a comment. I don't have any strong opinion about it

```suggestion
    # -- Reference to custom Issuer.
    # If generateIssuer is false, the issuer reference is required
    issuer:
      name: foo-org-ca
      kind: ClusterIssuer
      group: cert-manager.io
```

OR:
```suggestion
    # -- Reference to custom Issuer.
    # If generateIssuer is false, the issuer is required: 
    # { "issuer": {"name": "foo-org-ca", "kind": "ClusterIssuer", "group": "cert-manager.io" } }
    issuer: {}
      # name: foo-org-ca
      # kind: ClusterIssuer
      # group: cert-manager.io
```

WDYT @zroubalik ?